lib/metric.js

```diff
@@ -30,13 +30,18 @@ const Metric = class Metric {
 		if (!description) throw new Error('The argument "description" must be provided');
 
 		// Validation
-		if (!is.validDescription(description)) throw new Error('Provided value to argument "description" is not legal');
-		if (!is.validTimestamp(timestamp)) throw new Error('Provided value to argument "timestamp" is not legal');
-		if (!is.validSource(source)) throw new Error('Provided value to argument "source" is not legal');
-		if (!is.validLabels(labels)) throw new Error('Provided value to argument "label" is not legal');
-		if (!is.validValue(value)) throw new Error('Provided value to argument "value" is not legal');
-		if (!is.validType(type)) throw new Error('Provided value to argument "type" is not legal');
-		if (!is.validName(name)) throw new Error('Provided value to argument "name" is not legal');
+		if (!is.validDescription(description))
+			throw new Error(`Argument "description" must be a string, was '${typeof description}'`);
+		if (!is.validTimestamp(timestamp))
+			throw new Error(`Argument "timestamp" must be a finite number, was '${timestamp}'`);
+		if (!is.validSource(source)) throw new Error(`Argument "source" must be a string, was '${source}'`);
```

Comment:
Do we want a `typeof` here same as L34?
```suggestion
		if (!is.validSource(source)) throw new Error(`Argument "source" must be a string, was '${typeof source}'`);
```

Reply:
Agreed, it should be the same ❤️